hadoop-hdds/docs/content/interface/ReconApi.md

```diff
@@ -53,7 +53,7 @@ state of an Ozone cluster and to troubleshoot if needed.
     
     Returns all the ContainerMetadata objects.
     
-    ```json
+```json
```

Comment:
Good point. 

This is the result when I use IDE instead of CLI ;-) Fixed now with 

```
sed -i -E 's/[[:space:]]+```/```/g' ReconApi.zh.md`
```

But after I tested with `hugo serve` I found that the rendering still not perfect. Section titles are hard to identify/read:

![image](https://user-images.githubusercontent.com/170549/117798820-cc9f0200-b251-11eb-8f50-2b59f1cad48a.png)

Here it's not clear that 'containers/unhealthy is a section title'

https://ozone.apache.org/docs/1.1.0/interface/reconapi.html 

I fixed this styling, too. And now it seems to be more readable: 

![image](https://user-images.githubusercontent.com/170549/117798613-9792af80-b251-11eb-94a8-0f9c46ce8e89.png)